providers/ruby.rb

```diff
@@ -81,8 +81,6 @@ def perform_install
 
     Chef::Log.debug("#{new_resource} build time was " \
       "#{(Time.now - install_start) / 60.0} minutes")
-
-    new_resource.updated_by_last_action(true)
```

Comment:
You could do it now or we could clean it up later, but the this whole thing should get added to a converge_by block so we get nice messaging. Right now we're using the Log class do to this which isn't quite as nice.